src/components/SequenceInput/SequenceInput.js

```diff
@@ -0,0 +1,126 @@
+import React from 'react'
+import styled from 'styled-components/macro'
+import TimecodeInput from '../TimecodeTracker/TimecodeInput'
+import PropTypes from 'prop-types'
+
+SequenceInput.propTypes = {
+  hasDescription: PropTypes.bool,
+  hasTimeCode: PropTypes.bool,
+  inputValue: PropTypes.string,
+  isEmpty: PropTypes.bool,
+  onChange: PropTypes.func,
+  onDeleteClick: PropTypes.func,
+  onSaveClick: PropTypes.func,
+  setHasDescription: PropTypes.func,
+}
+
+export default function SequenceInput({
+  hasDescription,
+  hasTimeCode,
+  inputValue,
+  isEmpty,
+  onChange,
+  onDeleteClick,
+  onSaveClick,
+  setHasDescription,
+  textAreaValue,
+}) {
+  return (
+    <Wrapper isEmpty={isEmpty}>
+      <form onSubmit={onSaveClick}>
+        <Szenenbeschreibung
+          onChange={(event) =>
+            event.target.value.length > 0 && setHasDescription(true)
+          }
+          placeholder="Neue Szene hinzufügen"
+          name="description"
+          value={textAreaValue && textAreaValue}
+        ></Szenenbeschreibung>
+        {isEmpty && !hasDescription ? (
+          <Info>Szenenbeschreibung fehlt</Info>
+        ) : (
+          <Info hasDescription={hasDescription}>_</Info>
+        )}
+        <TimecodeInput
```

Comment:
I think you should name this TimeCodeInput with the C in camel case, since you are using camel case for TimeCode in other areas.

Reply:
Had this discussion [here](https://github.com/mpagels/capstone-project/pull/15#discussion_r480166395).
I stand by it.

Reply:
I understand your point and think this is fine as well. It just looks very weird to my eyes.